bin/generate.js

```diff
@@ -378,8 +378,8 @@ Creating a Design System People Love.
       process.stdout.write(`\nRebuilding node-sass`);
       loadingLoop(() => isNodeSassRebuilt);
       try {
-        await exec('npm rebuild node-sass', { cwd: params.dir });
-        log(chalk.green('\nSuccessfully rebuilt node-sass!'));
+        await exec('npm rebuild sass', { cwd: params.dir });
```

Comment:
I am pretty sure we don't need this anymore. This was an issue with node-sass where it would crap out if the wrong version of binaries were installed. 

I am not hearing that this is an issue with dart-sass.

Reply:
Sounds like we probably don't need this whole section then.

```javascript
      process.stdout.write(`\nRebuilding sass`);
      loadingLoop(() => isNodeSassRebuilt);
      try {
        await exec('npm rebuild sass', { cwd: params.dir });
        log(chalk.green('\nSuccessfully rebuilt sass!'));
      } catch ({ message }) {
        log(chalk.red(message));
      }
      isNodeSassRebuilt = true;
```

As well as cleaning up everywhere that `isNodeSassRebuilt` is referenced.

Sound right to you, @blackfalcon?

Reply:
yeop